README.md

```diff
@@ -57,7 +57,7 @@ To get started contributing to the project, see the [contributing guide](CONTRIB
 
 ## Documentation and demos
 
-The official documentation is hosted on [ReadTheDocs](https://docs.godotengine.org).
+The official documentation is hosted on [Read The Docs](https://docs.godotengine.org).
```

Comment:
ReadTheDocs is a website, so I think it's better to leave it as it is.

Reply:
The official spelling is ["Read the Docs"](https://readthedocs.org/) (case-sensitive).

Reply:
> The official spelling is ["Read the Docs"](https://readthedocs.org/) (case-sensitive), not "Read The Docs".

Ok thanks for the info will update the proposal.